docs/best-practice/worker-head-reconnection.md

```diff
@@ -20,11 +20,12 @@ https://github.com/ray-project/ray/blob/98be9fb5e08befbd6cac3ffbcaa477c5117b0eef
 
 It retries 600 times and each interval is 1s, resulting in total 600s timeout, i.e. 10 min. So immediately after 10-min wait for retries, each client exits and gets restarted while connecting to the new head IP. This issue exists in all stable ray versions (including 1.9.1). This has been reduced to 60s in recent commit in master. 
 
-## Best Practice
+## Solution
```

Comment:
We still have several things that need to be improved for GCS FT. Maybe we still mark it as alpha, and we will try to make this feature stable in the next release. Is it reasonable for you?

Reply:
I think this feature is more stable now and maybe we can mark as beta and for the next release stable. This feature is owned by @wilsonwang371, WDYT?

Reply:
Mark as beta SGTM